assets/app/styles/_components.less

```diff
@@ -202,40 +201,69 @@
       .osc-object-highlight();
       .flex(@columns: 1 1 auto);
       max-width: 450px;
+      min-width: 90px;
       border-radius: @border-radius-md;
-      border: 1px solid transparent;
-      // box-shadow: @osc-object-box-shadow; <- remove if it's not used in highlight
+      border: 1px solid #fff;
```

Comment:
why white instead of transparent? I think that may look odd on pages where the pods are on a background color (like the pods page)